llmfoundry/callbacks/eval_gauntlet_callback.py

```diff
@@ -120,6 +129,17 @@ def compute_averages(self, state: State) -> Dict[str, float]:
 
         return {k: sum(v) / len(v) for k, v in results.items()}
 
+    def calculate_averages(self, composite_scores):
```

Comment:
this has a confusingly similar name to `compute_averages`, could we add some explanation?

Also I think the other methods have type hints, `composite_scores: Dict[str, float]`?

Reply:
May also want to avoid a method that has the side effect of updating composite_scores, and also requires it be a class method. May be cleaner to make this a static method that just does one thing -- compute the named averages:
```
named_averages = compute_named_averages(composite_scores, averages)
composite_scores.update(named_averages)
```

by the way, this still has an issue if the `averages` contains the same name as a category in composite_scores, it gets overridden, so we need some protection there?

Maybe the named averages should be stored in a separate variable name?